electron/main.js

```diff
@@ -221,7 +224,7 @@ function showMainWindow() {
   });
 
   main.webContents.on('crashed', function() {
-    main.reload();
+    process.crash();
```

Comment:
Oops. You're right. This is the product of a sloppy merge from upstream on my part. I'll revert it.